### PR TITLE
New bio for GitHub Teacher

### PR DESCRIPTION
**Pull request by githubteacher**
Please help me decide what to tell the world!

@PatrickBM 
@githubschool/foundations-april

### CODE REVIEW

**Comment by pauljeffreys** [comment]
I have 2 cats :D

**Comment by cnmontreal** [comment]
hi

**Comment by HimanshuKeskar1** [comment]
This is going too fast

**Comment by githubteacher** [comment]
I think you should talk about your favorite animal.

**Comment by kkenyon** [comment]
My favorite animal  is a cat.

**Comment by t3hbeowulf** [comment]
For @HimanshuKeskar1 - https://chrome.google.com/webstore/detail/super-auto-refresh/kkhjakkgopekjlempoplnjclgedabddk?utm_source=chrome-app-launcher-info-dialog

**Comment by melissadabucon** [comment]
Favorite animal? Corgis.

**Comment by t3hbeowulf** [comment]
My favorite number is "B".  

o.O

File: bios/github-teacher.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This is my bio page.
+
+- My favorite color is blue.
```
Comment (githubteacher) [inline bios/github-teacher.md]: Exactly what shade is your favorite?

File: bios/github-teacher.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+- My favorite food is Italian.
```
Comment (cnmontreal) [inline bios/github-teacher.md:4]: favorite food

**Comment by pauljeffreys** [comment]
:+1:

**Comment by pauljeffreys** [comment]
I think I just closed...

**Comment by githubteacher** [comment]
Made this change for #29

**Comment by rezwang** [comment]
only fish

**Comment by cnmontreal** [comment]
made this change for #120

**Pull request merged**